src/domain/Filters/FilterController/FilterController.tsx

```diff
@@ -120,4 +134,17 @@ export class FilterController extends React.PureComponent<IFilterControllerProps
       </InputGroup.Button>
     )
   }
+
+  private filterButtonWithRoundBorderAllSide = ({ toggleMenu, toggleComponentRef, ariaProps }: any) => {
```

Comment:
can probably rename this function to `filterButton` and the other one to `inputGroupFilterButton`